packages/cli/src/commands/up.ts

```diff
@@ -119,14 +121,23 @@ export default class Up extends MachineCreationDriverCommand<typeof Up> {
 
     const flatTunnels = flattenTunnels(tunnels)
 
+    const result = await asyncReduce(
+      { urls: flatTunnels },
+      async ({ urls }, envCreated) => envCreated(
+        { log: this.logger, userModel: await this.ensureUserModel() },
```

Comment:
ensureUserModel() is used on every iteration, is that the right behavior?

Reply:
it's cached (but fixed anyway)